sw/device/tests/BUILD

```diff
@@ -3604,6 +3604,76 @@ opentitan_test(
     ],
 )
 
+# This test requires that the harness has sufficient permissions to
+# open the USB parent hub of the device under test so it can issue
+# requests directly. For this reason, the test is tagged as manual
+# until the CI can provide such an access.
+opentitan_test(
+    name = "usbdev_aon_wake_reset_test",
```

Comment:
I clicked approve a little bit too fast. This already exists in the BUILD file.

Reply:
good catch!